proto/foxglove/CompressedImage.proto

```diff
@@ -0,0 +1,11 @@
+// https://foxglove.dev/docs/studio/messages/compressed-image
```

Comment:
It would probably be ideal to have the full docs in here as comments, so the message files are really self-contained